doc/source/tutorial/stats.rst

```diff
@@ -102,7 +102,7 @@ introspection:
     >>> dist_discrete = [d for d in dir(stats) if
     ...                  isinstance(getattr(stats, d), stats.rv_discrete)]
     >>> print('number of continuous distributions: %d' % len(dist_continu))
-    number of continuous distributions: 96
+    number of continuous distributions: 97
```

Comment:
This number 97 will need to change after a rebase.

Reply:
Thanks for the revision.